pkg/project/project.go

```diff
@@ -1,13 +1,18 @@
 package project
 
 var (
-	description        = "The aws-operator handles Kubernetes clusters running on a Kubernetes cluster inside of AWS."
-	gitSHA             = "n/a"
-	name        string = "aws-operator"
-	source      string = "https://github.com/giantswarm/aws-operator"
-	version            = "n/a"
+	bundleVersion        = "6.4.0"
```

Comment:
Just some food for thought. Not blocking this PR. Thinking out "loud" here. Can't we inject this via `architect` in the future? Maybe just having the version in the chart values. Then we can put it where ever we need it. Also into the labels. Then we know the version at deployment time and can solve the issue with duplicated reconciliation using an admission controller.

Reply:
We're all on the same page here, I think :D

Reply:
We're all on the same page here, I think :D

Reply:
Yes we are heading towards this, let's have this reflection in the team.